code/midtown/mmgame/multicr.cpp

```diff
@@ -51,3 +61,136 @@ i32 mmMultiCR::SelectTeams()
 
 void mmMultiCR::SendSetup(ulong /*arg1*/)
 {}
+
+void mmMultiCR::StealGold(mmCar* car)
+{
+    GoldCarrier = car;
+
+    mmMultiCR::FondleCarMass(GoldCarrier, float(MMSTATE.CRGoldMass));
+    field_1EEB0->Initialized = true;
+}
+
+void mmMultiCR::UpdateGame()
+{
+    switch (GameState)
+    {
+        case 0:
+            if (NETMGR.IsHost())
+                GameState = 1;
+            break;
+
+        case 1:
+            GameState = 2;
+            if (MMSTATE.HasMidtownCD)
+                AudMgr()->PlayCDTrack(GetCDTrack(10), true);
+            Player->Hud.StopTimers();
+            break;
+
+        case 2:
+            if (MMSTATE.CRLimitMode == mmCRLimitMode::Time && NETMGR.IsHost())
+            {
+                field_1EEFC.Start();
+                Player->Hud.BlitzTimer.Start();
+            }
+
+            StartSounds->ActiveSound = 0;
+            StartSounds->PlayOnce(-1.0, -1.0);
+
+            Player->Hud.SetMessage(AngelReadString(0x83u), 2.0f, true);
+
+            if (VoiceCommentary)
+                VoiceCommentary->PlayCRPreRace();
+
+            EnableRacers();
+            GameState = 4;
+            break;
+
+        case 4:
+            if (!MMSTATE.DisableDamage && Player->IsMaxDamaged())
+            {
+                Player->Hud.SetMessage(AngelReadString(0x84u), 5.0f, false);
```

Comment:
Use `LOC_STR(MM_IDS_184)` (but also rename MM_IDS_184 to something more descriptive)